Modules/Core/Support/EInvoiceHelper.php

```
@@ -128,14 +127,14 @@ public static function get_xml_full_name(string $xml_id)
+        $query = \Modules\Core\Models\User::where('user_type', '1')
+            ->where('user_active', '1');
 
-        $where = ['user_type' => '1', 'user_active' => '1'];
         if ($user_id) {
-            $where['user_id'] = $user_id;
+            $query->where('user_id', $user_id);
         }
 
-        return $CI->db->from('ip_users')->where($where)->get()->result();
+        return $query->get()->toArray();
     }
```

PHPMD (2.15.0)

The method get_admin_active_users is not named in camelCase. (undefined)

(CamelCaseMethodName)